modules/libcom/src/fdmgr/fdManager.cpp

```diff
@@ -31,27 +31,41 @@ using std :: max;
 
 fdManager fileDescriptorManager;
 
-const unsigned mSecPerSec = 1000u;
-const unsigned uSecPerSec = 1000u * mSecPerSec;
+static const unsigned mSecPerSec = 1000u;
+static const unsigned uSecPerSec = 1000u * mSecPerSec;
+
+#ifdef FDMGR_USE_POLL
+static const int PollEvents[] = { // must match fdRegType 
+    POLLRDNORM | POLLRDBAND | POLLIN | POLLHUP | POLLERR,
+    POLLWRBAND | POLLWRNORM | POLLOUT | POLLERR,
+    POLLPRI};
```

Comment:
I do not think it is desirable to request events absent specific handling.  If one of these unhandled events does occur, the result would be spinning in a tight loop.  eg. `POLLRDBAND` and `POLLWRBAND` are not handled by a regular `recv()`.

Reply:
fyi. how libevent maps `poll()` events https://github.com/libevent/libevent/blob/9a6a799e7002aae09d5e7cb244bdb913eed75a04/poll.c#L212-L220 .

Reply:
I took the mapping from select to poll from here:
https://www.man7.org/linux/man-pages/man2/select.2.html
See under „Notes“ the paragraph „ Correspondence between select() and poll() notifications“.

Reply:
Fair.  I guess we are looking for bug-for-bug compatibility...